README.md

```diff
@@ -3,201 +3,180 @@
-the top level meta feed or a base64 encoded nonce. Feedformat can be
-either `bendy butt` for a meta feed or `classic`.
+Now you can create subfeeds *under* that root meta feed like this:
 
 ```js
-const seed = sbot.metafeeds.keys.generateSeed()
-const mfKey = sbot.metafeeds.keys.deriveFeedKeyFromSeed(seed, 'metafeed')
-```
-
-### messages
+const details = {
+  feedpurpose: 'mygame',
+  feedformat: 'classic',
+  metadata: {
+    score: 0,
+    whateverElse: true
+  }
+}
 
-Low level api for generating messages
+sbot.metafeeds.create(metafeed, details, (err, subfeed) => {
+  console.log(subfeed)
+  // {
+  //   feedpurpose: 'mygame',
+  //   subfeed: '@___________________________________________=.ed25519',
+  //   keys: {
+  //     curve,
+  //     public,
+  //     private,
+  //     id
+  //   }
+  // }
+})
+```
 
-#### addExistingFeed(metafeedKeys, previous, feedpurpose, feedKeys, metadata)
+This has created a `keys` object for a new subfeed, which you can use to publish
+application-specific messages (such as for a game). The `details` argument always
+needs `feedpurpose` and `feedformat` (supports `classic` for ed25519 normal SSB
+feeds, and `bendy butt`).
 
-Generate a message linking an existing feed to a meta feed. `Previous`
-is the previous message on the meta feed in key value form. `metatada`
-is an optional dict.
+To look up sub feeds belonging to the root meta feed, we can use `filter` and `find`. These are similar to Array [filter](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter) and [find](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find):
 
 ```js
-const msg = sbot.metafeeds.messages.addExistingFeed(metafeedKeys, null, 'main', mainKeys)
+sbot.metafeeds.filter(metafeed, f => f.feedpurpose === 'mygame', (err, feeds) => {
+  console.log(feeds)
+  // [
+  //   { feedpurpose, subfeed, keys }
+  // ]
+})
 ```
 
-#### addNewFeed(metafeedKeys, previous, feedpurpose, seed, feedformat, metadata)
-
-Generate a message to be posted on meta feed linking feed to a meta
-feed. Similar to `deriveFeedKeyFromSeed`, `feedformat` can be either
-`bendy butt` for a meta feed or `classic`. `metatada` is an optional
-dict.
-
 ```js
-const msg = sbot.metafeeds.messages.addNewFeed(metafeedKeys, null, 'main', seed, 'classic')
+sbot.metafeeds.find(metafeed, f => f.feedpurpose === 'mygame', (err, feed) => {
+  console.log(feed)
+  // { feedpurpose, subfeed, keys }
+})
 ```
 
-#### tombstoneFeed(metafeedKeys, previous, feedKeys, reason, cb)
+Notice that each `f` (and the result) is an object that describes the subfeed, with the fields:
 
-Generate a message to be posted on meta feed tombstoning a feed on a
-meta feed. `Previous` is the previous message on the meta feed in key
-value form.
+* `feedpurpose`: same string as used to create the subfeed
+* `subfeed`: the SSB identifier for this subfeed
+* `keys`: the [ssb-keys] compatible `{ curve, public, private, id }` object
```

Comment:
I'll read the code to see what it does, but from just reading the README I was thinking it would also have whatever metadata you have attached as in the example with the game?

Reply:
Done!